clang/lib/CodeGen/CodeGenModule.cpp

```diff
@@ -31,6 +31,7 @@
 #include "clang/AST/DeclCXX.h"
 #include "clang/AST/DeclObjC.h"
 #include "clang/AST/DeclTemplate.h"
+#include "clang/AST/Expr.h"
```

Comment:
Could you double check that this include is needed?

Reply:
Not needed.